source/administration/devops-command-center.rst

```diff
@@ -64,48 +64,126 @@ Viewing and creating playbooks
 
 Select **!** in the channel header to open the RHS panel. Select the **Playbooks** icon to open the playbooks backstage and list all playbooks associated with the current team. Select **+ New Playbook** to start a new playbook and checklist. Enter a playbook name and choose **Save Playbook**.
 
-Creating checklists
-~~~~~~~~~~~~~~~~~~~
+Public and private incidents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Checklists are created within a playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a playbook, or during an incident.
+Incidents can be:
 
-Editing checklists
-~~~~~~~~~~~~~~~~~~~
+- **Public:** Anyone in the team can view the incident details and join the incident channel.
+- **Private:** Only incident members who were in the incident when it was started have access to the ongoing incident and view incident details after the incident has ended.
```

Comment:
Nit: "Only incident members who were in the incident when it was started" may be a bit misleading, because you can invite people to the channel after the incident was started. Maybe we can reword it to something like
```suggestion
- **Private:** Only incident members who were members of the channel of the incident have access to the ongoing incident and view incident details after the incident has ended.
```
That came up a bit convoluted, even cumbersome, just wanted to showcase what I mean.